test/Garnet.test/GarnetClientTests.cs

```diff
@@ -575,5 +578,21 @@ private static async Task<long> DeleteKeysWithCT(string[] keys, Memory<byte>[] k
 
             return 0;
         }
+
+        [Test]
+        public async Task UnixSocketServer_Ping()
```

Comment:
This is a great start. I think we should add a few tests in a dedicated UnixSocketTests class, using SE.Redis as the client. As per this, unix sockets are supported by SE.Redis already: https://github.com/StackExchange/StackExchange.Redis/pull/850

Reply:
Added [UnixSocketTests.cs](https://github.com/microsoft/garnet/pull/944/files#diff-62ec26dcbe948c7dd740cd2d06d71f3b49902656daaee80b30e0d137fe62fbf5) which is `SE.Redis` based. I'm leaving the single unix socket test `GarnetClientTests` as `GarnetClient` specific test.